src/process-analysis/suggestions.tex

```diff
@@ -1,5 +1,13 @@
-% textidote: ignore begin
-
-
+%textidote: ignore begin
 \section{Suggestions for P3}\label{sec:suggestions-for-p3}
-% textidote: ignore end
+%textidote: ignore end
+Suggestions for future projects would be to make the group work more spread out and more evenly distributed.
+This way the workload would be more balanced, and the group would be able to work more efficiently.
+We could also suggest that the group have more regular meetings and present their progress to the group, this
+could be a presentation of some code the specific group member had written.
+The group also had a problem with the problem statement, as we did not do enough research on how we could solve the
+problem with the current technology of today.
+
+Other than that, there are not a lot of suggestions for future projects,
+as the group has worked well together and have been able to solve the problems
+that had unfolded during the project period.
```

Comment:
I think we should delete these sentences, as they are concluding, which should probably just be done in the conclusion, and because they suggest that are not a lot of possible suggestions, which is a little naive.